components/view/view.js

```diff
@@ -0,0 +1,72 @@
+import { default as UTILS } from '/js/deer-utils.js'
+import { default as DEER } from '/js/deer-config.js'
+
+const template = (obj, options = {}) => {
+    let indent = options.indent ?? 4
+    let replacer = (k, v) => {
+        if (DEER.SUPPRESS.indexOf(k) !== -1) return
+        return v
+    }
+    try {
+        return `<pre>${JSON.stringify(obj, replacer, indent)}</pre>`
```

Comment:
You no longer have a `prop` template because it was in deer-render.js, and so this is where it defaults.  I think it is drawing more than you want it to.

![image](https://user-images.githubusercontent.com/3287006/185476235-1c94f40a-0389-44d5-aaeb-022bdad5b9e0.png)

Reply:
next PR has it